### PR TITLE
Expose epoch duration for AURA for polkadotjs-api

### PR DESCRIPTION
**Pull request by drewstone**
@jacogr maybe this is useful for the `api-derive` call around unbonding times. We can use this parameter to mimic the session count per era or the respectively parameter that is used in BABE as an `EPOCH`.

### CODE REVIEW

**Comment by bkchr** [comment]
Why do you need this?

It makes no sense to add a parameter, that is not used :shrug:

**Comment by drewstone** [comment]
Ok, happy to close. It came up in conversation with @jacogr because the UI is displaying unbonding times without this parameter.

**Comment by jacogr** [comment]
So where it does make sense is the palette it is actually used in, i.e. where the sessionLength is determined. Which is session. So originally, pre-migration, it was on the `sessionLength` storage item. That was removed and Babe (as it determines the length), exposes that constant.

If it is do-able on session, have not looked.

**Comment by andresilva** [comment]
In principle I agree with @jacogr that it should be exposed by whoever defines it. In the case of BABE the epoch length is determined by itself and hence it is exposed there. The problem is the session module doesn't have any notion of session length and it just delegates it to a trait [`ShouldEndSession`](https://github.com/paritytech/substrate/blob/master/frame/session/src/lib.rs#L131) and Edgeware probably uses [`PeriodicSession`](https://github.com/paritytech/substrate/blob/master/frame/session/src/lib.rs#L139). So I'm not sure where this should be exposed.

File: frame/aura/src/lib.rs [hunk 1]
```diff
@@ -86,7 +89,9 @@ decl_storage! {
 }
 
 decl_module! {
-	pub struct Module<T: Trait> for enum Call where origin: T::Origin { }
+	pub struct Module<T: Trait> for enum Call where origin: T::Origin {
+		const EpochDuration: EraIndex = T::Period::get();
```
Comment (jacogr) [inline frame/aura/src/lib.rs:93]: I believe the "epoch" wording is babe-specific, would just call it "SessionDuration" (or "SessionLength", whatever makes sense in the context it is used in)

**Pull request closed without merging**